velox/expression/tests/CMakeLists.txt

```diff
@@ -88,8 +90,9 @@ target_link_libraries(
   velox_expression_verifier velox_vector_test_lib velox_vector_fuzzer
   velox_type velox_expression_test_utility)
 
-add_library(velox_expression_fuzzer ExpressionFuzzer.cpp FuzzerRunner.cpp
-                                    ExpressionFuzzerVerifier.cpp)
+add_library(
```

Comment:
I'm thinking that it might be nicer to extract Fuzzer into a separate directory, maybe, velox/expression/fuzzer.

Reply:
Filed https://github.com/facebookincubator/velox/pull/9651 to extract fuzzer related code to a separate directory.